docs/00-introduction.md

```diff
@@ -0,0 +1,75 @@
+git config --global core.autocrlf input # <-- macOS, Linux only
+git config --global core.autocrlf true  # <-- Windows only
+git config --global pull.rebase true
+git config --global core.ignorecase false
+```
+
+- Create a **local directory** that you reserve for everything related to this course. Within this directory, you can clone your GitHub repository. On your repository's GitHub page, click on the green button on the right (Code), choose "Use SSH", and copy the link in the format `git@github.com:HoGentTIN/REPO_NAME-USERNAME.git`. This will create a local copy of the repository in a subdirectory. You can rename this directory and move it; everything will still work. It's a bad idea to clone the repository in a folder that is synced to the cloud (e.g., OneDrive, Dropbox, Google Drive, iCloud, etc.).
+
+```bash
+cd Documents/Courses/MLOps/
+git clone git@github.com:HOGENT-MLOps/REPO_NAME-USERNAME.git
+```
+
+- Review the files in the `assignments` directory. Here you'll find the assignments for all labs.
+- In the `reports` directory, you'll find templates for reports and cheat sheets in [Markdown format](https://guides.github.com/features/mastering-markdown/). Customize the template by adding your name.
+    - There is an example assignment `00-dummy.md` with an example report `00-dummy-report.md`. Have a look at them to see how you are expected to structure your lab reports.
+- Also, immediately add your name and repository URL to the `README.md`.
+
+## Processing errata
+
+It is possible that errors may be found in the lab assignments during the semester. These will be published in the [source repository](https://github.com/HOGENT-MLOps/mlops-labs). You can systematically apply these changes to your own personal GitHub repository where you are working on the lab assignments. Simply open your GitHub repository in the browser and click the "Sync" button.
+
+Any errata will be announced via Chamilo.
+Hier is de vertaling van de HTML-pagina naar het Engels en omgezet in Markdown:
+
+## General Guidelines
+
+In this section, you will find some general guidelines to help you work more smoothly and efficiently.
+
+- Before starting a lab assignment, prepare yourself by studying all the provided study materials: manuals, screencasts, etc. There is no need to "memorize" everything, but make sure you are familiar enough with the content so that you can quickly find the correct and relevant information. You can find this through the references in the individual lessons on this page.
+
+- **Work step by step.** Don't write too much code at once. First, try to obtain a minimally working solution and immediately commit it to Git. Make minimal changes and test each change thoroughly. The larger and more significant the changes, the greater the chance of errors and the more difficult they are to debug. As soon as something works and you've made progress, commit it to Git right away with a clear, descriptive commit message.
```

Comment:
Specification for commit messages: https://www.conventionalcommits.org/en/v1.0.0/